services/graph/pkg/unifiedrole/unifiedrole.go

```diff
@@ -476,6 +505,8 @@ func displayName(role *conversions.Role) *string {
 		displayName = "Can upload"
 	case conversions.RoleManager:
 		displayName = "Can manage"
+	case conversions.RoleSecureViewer:
+		displayName = "Secure View"
```

Comment:
@tbsbdr  I could be wrong, but AFAIK you came up with all the other roles names. Are you ok with "Secure View" as then display name of the role?

Give that the other roles have display names like "Can upload", "Can view" and "Can edit" it seems to me that "Secure View" is somewhat inconsistent.

Also the difference between "Can view" and "Secure View" is not really obvious to the user I think.

Reply:
hm yes, good point.
Least bad proposal is:

- Can view (secure)
- Can view
- Can edit